lib/katello/engine.rb

```diff
@@ -26,6 +26,7 @@ class Engine < ::Rails::Engine
         :redhat_repository_url => 'https://cdn.redhat.com',
         :consumer_cert_rpm => 'katello-ca-consumer-latest.noarch.rpm',
         :consumer_cert_sh => 'katello-rhsm-consumer',
+        :event_daemon_enabled => true,
```

Comment:
What happens when I set this to false? Is there a occasion to do that?

Reply:
It doesn't run the event listeners. It could be useful if you want to create a backlog of events for load testing, or another reason we'll inevitably run into!